index.html

```diff
@@ -205,87 +240,87 @@ <h2 class="card-title">Tonic</h2>
-            <a href="https://twitter.com/HFG__43"
-              ><img src="Images_Background/Twitter.svg" alt="Twitter Icon"
-            /></a>
-          </li>
-          <li>
-            <a href="#"><img src="Images_Background/M.svg" alt="M Icon" /></a>
-          </li>
-        </ul>
-        <div class="button-container principal-margin">
-          <button class="see-project-button">Get my resume</button>
-        </div>
-        <ul class="about-myself-skills principal-margin">
-          <li class="arrow-skills">
-            <p>Languages</p>
-            <i class="fa fa-angle-down arrow-color"></i>
-          </li>
-          <ul class="tech-languages">
+          </p>
+          <h2 class="lets-connect principal-margin-about">LET'S CONNECT</h2>
+          <ul class="social-icons social-icons-margin-bottom-about-myself principal-margin-about-social">
+            <li>
+              <a href="https://github.com/HFG43"
+                ><img src="Images_Background/GitHub.svg" alt="GitHub Icon"
+              /></a>
+            </li>
             <li>
-              <img src="Images_Background/JS-logo.svg" alt="JavaScript logo" />
-              <p class="language-name">JavaScript</p>
+              <a
+                href="https://www.linkedin.com/in/hern%C3%A1n-g%C3%BCemes-a440591b/"
+                ><img
+                  src="Images_Background/Linkedin icon.svg"
+                  alt="LinkedIn Icon"
+              /></a>
             </li>
             <li>
-              <img src="Images_Background/html-logo.svg" alt="HTML logo" />
-              <p class="language-name">HTML</p>
+              <a href="#"
+                ><img src="Images_Background/Fingers.svg" alt="Fingers Icon"
+              /></a>
             </li>
             <li>
-              <img src="Images_Background/css-logo.svg" alt="CSS logo" />
-              <p class="language-name">CSS</p>
+              <a href="https://twitter.com/HFG__43"
+                ><img src="Images_Background/Twitter.svg" alt="Twitter Icon"
+              /></a>
+            </li>
+            <li>
+              <a href="#"><img src="Images_Background/M.svg" alt="M Icon" /></a>
             </li>
           </ul>
-          <li class="bottom-line-language arrow-skills">
-            <p>Frameworks</p>
-            <i class="fa fa-angle-down arrow-color"></i>
-          </li>
-          <li class="bottom-line-language arrow-skills">
-            <p>Skills</p>
-            <i class="fa fa-angle-down arrow-color"></i>
-          </li>
-        </ul>
+          <div class="button-container principal-margin-about">
+            <button class="see-project-button">Get my resume</button>
+          </div>
+        </div> 
+        <div class="about-myself-skills-section">
+          <ul class="about-myself-skills principal-margin">
+            <li class="arrow-skills">
+              <p>Languages</p>
+              <i class="fa fa-angle-down arrow-color"></i>
+            </li>
```

Comment:
- [ ] on the mobile version the languages section is not aligned and spaced well like the figma image below please kindly make the necessary corrections. :+1: 
- figma design
![l-m-f](https://user-images.githubusercontent.com/89055502/236370371-fd7c6188-2c6d-445a-872d-15a9ef026586.png)
-your design
![l-m-s](https://user-images.githubusercontent.com/89055502/236370396-8bcc7342-d38b-48cd-9f6a-ce7b584f8bec.png)

Reply:
@Efiamotu-1 Many thanks Habeeb Efiamotu, I really appreciate your feedback! 🙌🙌
I will work on these corrections!